bots/pr/src/test/java/org/openjdk/skara/bots/pr/ApprovalTests.java

```diff
@@ -31,19 +31,20 @@
 public class ApprovalTests {
     @Test
     void simple() {
-        Approval approval = new Approval("", "jdk17u-fix-request", "jdk17u-fix-yes", "jdk17u-fix-no");
+        Approval approval = new Approval("", "jdk17u-fix-request", "jdk17u-fix-yes", "jdk17u-fix-no", "https://test.openjdk.org");
```

Comment:
I recommend using something with `example.com` for test URL.

Reply:
Sure, will fix it.